rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -166,9 +166,29 @@ def test_reconciliation_query(self):
     self._insert_samples(p_on_time, BIOBANK_TESTS[:2], ['GoodSample1', 'GoodSample2'], 'OGoodOrder',
                          within_24_hours, within_24_hours - datetime.timedelta(hours=1))
 
+
+
+    # On time, recent order and samples not confirmed shows up in missing;
+    p_unconfirmed_samples = self._insert_participant()
+    self._insert_order(p_unconfirmed_samples, 'not_confirmed_order', BIOBANK_TESTS[:4], order_time,
```

Comment:
Do these actually show up in missing? It hasn't been 24 hours.... so I'm not sure they should.

Reply:
corrected error in insert, good catch